storage/src/backend/connection.rs

```diff
@@ -290,7 +306,13 @@ impl Connection {
                 // fallback to origin server, the policy only applicable to non-upload operation
                 warn!("Request proxy server failed, fallback to original server");
             } else {
-                warn!("Proxy server is not healthy, fallback to original server");
+                LAST_LOGGING_TIME.with(|f| {
+                    let current = SystemTime::now();
+                    if current.duration_since(*f.borrow()).unwrap().as_secs() >= 2 {
+                        warn!("Proxy server is not healthy, fallback to original server");
```

Comment:
Is it possible to get all responses from the original server within 2 seconds, then without printing any warning?

Reply:
The proxy is still unhealthy, so the warning can be printed after 2s. To ensure the first warning must be printed. I can make LAST_LOGGING_TIME start from EPOCH.